terraform/locals.tf

```diff
@@ -0,0 +1,9 @@
+locals {
+  api_service_backend_url_options = {
+    uk = "https://${var.base_domain}/uk/"
+    xi = "https://${var.base_domain}/xi"
+  }
+
+  # TODO: Revisit this when PaaS is no more
```

Comment:
So this is actually a hard requirement coming from the plek gem which is a nested dependency of one of our other govuk gems https://github.com/alphagov/plek

Reply:
So we'll never get rid of it :(

Reply:
Makes sense. I'll remove the TODO 😢